components/SingleVcItem.tsx

```diff
@@ -29,27 +29,6 @@ const VerifiedIcon: React.FC = () => {
 };
 
 const getDetails = (arg1, arg2, verifiableCredential) => {
-  if (arg1 === 'Full Name') {
```

Comment:
Why is this branch removed? I see it being used later in line#166, is this code change required to remove the radio-button select element and the loading icon which came underneath it?

Reply:
This code is removed to keep it consistent with the VCitem.tsx. In that component they are checking the condition for status and if it is status lines allotted are 1. And for other it is 4. Only for status it is changing and all others are same. So i have confirmed with vijay that this condition need not be checked and it is a redundant code and it can be removed and there are no issues.